network/interface.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2020. Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package network
+
+import (
+	"github.com/nuts-foundation/nuts-node/network/rfc004"
+	"time"
+)
+
+// NetworkClient is the interface to be implemented by any remote or local client
+type NetworkClient interface {
```

Comment:
In Nuts-node client interfaces are no longer used. Just 1 interface for interaction between modules. So maybe just the name is misleading.

Reply:
Renamed to Network